crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -100,6 +100,11 @@ impl From<Color> for StandardMaterial {
     fn from(color: Color) -> Self {
         StandardMaterial {
             base_color: color,
+            alpha_mode: if color.a() != 1.0 {
```

Comment:
```suggestion
            alpha_mode: if color.a() < 1.0 {
```
?

Reply:
Yeah, that's probably better, this way it enforces the fact that alpha is in the 0..1 range.